.github/workflows/publish.yml

```diff
@@ -56,10 +57,74 @@ jobs:
 
           os.system(f'echo "increment={increment}" >> $GITHUB_OUTPUT')
           os.system(f'echo "version={ultralytics.__version__}" >> $GITHUB_OUTPUT')
+          os.system(f'echo "previous_version={check_latest_pypi_version()}" >> $GITHUB_OUTPUT')
 
           if increment:
               print('Local version is higher than PyPI version. Publishing new version to PyPI ✅.')
         id: check_pypi
+      - name: Summarize version changes
+        env:
+          OPENAI_AZURE_API_KEY: ${{ secrets.OPENAI_AZURE_API_KEY }} 
+          OPENAI_AZURE_ENDPOINT: ${{ secrets.OPENAI_AZURE_ENDPOINT }} 
```

Comment:
I need to find out which region these secrets refer to. Do you know who set them @glenn-jocher ?